doc/releases/release-notes-2.1.rst

```diff
@@ -89,28 +340,162 @@ Bluetooth
 
 * TBD
 
+16874 : Bluetooth: Allow disconnected state connections to be reconnected : jhedberg : Johan Hedberg
+
+18338 : bluetooth: controller: Added function to support RX link quota update : mtpr-ot : Morten Priess
+18477 : bluetooth: controller: Add vendor operations to TICKER_USER_ULL_HIGH_OPS : mtpr-ot : Morten Priess
+20254 : bluetooth: controller: Add vendor specific overhead to TX buffer size : mtpr-ot : Morten Priess
+18327 : bluetooth: controller: Connection meta property support in LLL : mtpr-ot : Morten Priess
+19657 : Bluetooth: controller: Handle Read Remote version as a priority event : carlescufi : Carles Cufí
+19240 : Bluetooth: controller: legacy: Ensure that a `case` statement is present : cvinayak : Vinayak Kariappa Chettimada
```

Comment:
The script that creates these lists of issues needs to scan for meaningful reST syntax and escape those so they're not interpreted by Sphinx.  In this case replace `` ` `` with ``\` ``  (or better would be to change `` ` `` (back-tick) to `` ' `` (apostrophe)